artefacts/pageContent_schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$id": "root",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "page": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Decentralised Identifier of the entity for which the PageContent will be made",
+          "format": "DID",
+          "examples": ["did:ixo:CYCc2xaJKrp8Yt947Nc6jdff"]
+        },
+        "@type": {
+          "type": "string",
+          "default": "ixo:page",
+          "description": "The type of entity attribute. As defined in the ixo Context library."
+        },
+        "page": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "Each of the page content ID which is generated by editorjs (random hash value)",
+                "examples": ["z3rRA9MDzL"]
+              },
+              "type": {
+                "type": "string",
+                "description": "Content types supported by editorjs",
+                "examples": [
+                  "paragraph",
+                  "table",
+                  "image",
+                  "list",
+                  "warning",
+                  "code",
+                  "rawHTML",
+                  "header",
+                  "quote",
+                  "checklist",
+                  "delimitter"
+                ]
+              },
+              "data": {
+                "type": "object",
```

Comment:
@ig-shaun 
`data` should be an any type as every contents has different values.

for example:
```
{
	id: "z3rRA9MDzL",
	type: "paragraph",
	data: {
		text: "Hello World!"
	}
}
```

vs

```
{
	id: "Y23Hu9teuc",
	type: "raw",
	data: {
		html: "<html>...</html>"
	}
}
```

Reply:
`data` is a linkedResource that will have the pattern:
```
{
    "type": "data",
    "id": "#fwefw0wu5-w0j5-wjr",  // the content identifier (cid) of the resource
    "description": "IoT Events Data",
    "mediaType": "json",
    "endpoint": "#cellnode/fwefw0wu5-w0j5-wjr",
    "proof": "fwefw0wu5-w0j5-wjr", // matches the cid in this instance, as the content hash serves as a proof
    "encrypted": "false"
  }
```